src/main.rs

```diff
@@ -51,10 +60,21 @@ fn main() {
                  .help("name of the fuzz target"))
             .arg(Arg::with_name("CORPUS").multiple(true)
                  .help("custom corpus directory or artefact files"))
-            .arg(Arg::with_name("ARGS").multiple(true).last(true)
-                 .help("additional libFuzzer arguments passed to the binary"))
             .arg(Arg::with_name("TRIPLE").long("target")
+                 .takes_value(true)
+                 .default_value(utils::default_target())
```

Comment:
not a big deal, but `default_value` implies `takes_value`

https://docs.rs/clap/2.24.2/clap/struct.Arg.html#method.default_value